hw/ip/aes/pre_sca/alma_post_syn/load_env.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This file needs to be sources into shell as follows:
+# `source load_env.sh`
+
+# The top folder for opentitan repository
+export REPO_TOP=~/Documents/opentitan
```

Comment:
The REPO_TOP variable could be re-used from `util/build_consts.sh`. I think I would change this script check that the variable REPO_TOP exists and if not, exit and ask the user to source `util/build_consts.sh`.